ghcide/src/Development/IDE/Core/UseStale.hs

```diff
@@ -1,162 +1,162 @@
-{-# LANGUAGE DerivingVia    #-}

```

Comment:
omg all those files had incorrect line endings???? :facepalm:

Reply:
yeah, as described in the issue: https://github.com/haskell/haskell-language-server/issues/2678